scripts/js/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 120,
+  "tabWidth": 4,
+  "bracketSpacing": true
+}

```

Comment:
It feels like we duplicate this config quite a lot. Same goes for .eslintrc.js. IDK if we want to have only one of them or multiple.